### PR TITLE
add rosbridge launch

### PR DESCRIPTION
**Pull request by yabuta**
## PR Type

- Improvement

## Related Links

https://github.com/autowarefoundation/autoware.universe/pull/318

## Description

Start rosbridge with api launch file.
And remove rosbridge launch from web_controller

## Review Procedure

- `ros2 launch autoware_api_launch autoware_api.launch.xml init_simulator_pose:=false init_localization_pose:=false`
- check `ros2 node list|grep rosbridge`

## Pre-Review Checklist for the PR Author

**PR Author should check the checkboxes below when creating the PR.**

- [x] Code follows [coding guidelines][coding-guidelines]
- [x] Assign PR to reviewer

## Checklist for the PR Reviewer

**Reviewers should check the checkboxes below before approval.**

- [x] Commits are properly organized and messages are according to the guideline
- [x] Code follows [coding guidelines][coding-guidelines]
- [ ] (Optional) Unit tests have been written for new behavior
- [x] PR title describes the changes

## Post-Review Checklist for the PR Author

**PR Author should check the checkboxes below before merging.**

- [ ] All open points are addressed and tracked via issues or tickets
- [ ] Write [release notes][release-notes]

## CI Checks

- **Build and test for PR**: Required to pass before the merge.
- **Check spelling**: NOT required to pass before the merge. It is up to the reviewer(s). See [here][spell-check-dict] if you want to add some words to the spell check dictionary.

[coding-guidelines]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/1194394777/T4
[release-notes]: https://tier4.atlassian.net/wiki/spaces/AIP/pages/563774416
[spell-check-dict]: https://github.com/tier4/autoware-spell-check-dict#how-to-contribute

### CODE REVIEW

File: autoware_api_launch/launch/autoware_api.launch.xml [hunk 1]
```diff
@@ -19,4 +21,10 @@
     <include file="$(find-pkg-share path_distance_calculator)/launch/path_distance_calculator.launch.xml"/>
   </group>
 
+  <!-- rosbridge -->
+  <!-- TODO:　respawn will work once rosbridge is fixed. I'll delete this comment when it's done. -->
```
Comment (kenji-miyake) [inline autoware_api_launch/launch/autoware_api.launch.xml]:
```suggestion
  <!-- TODO:　respawn will work once https://github.com/ros2/launch/pull/569 is released. I'll delete this comment when it's done. -->
```

**Review by kenji-miyake — DISMISSED** [review]

**Comment by yabuta** [comment]
またエラーが起きてる

**Review by kenji-miyake — APPROVED** [review]

**Comment by shmpwk** [comment]
@yabuta @kenji-miyake 
Do you know why rosbridge is launched in `autoware_api_launch`  but not in `web_controller`? 
We are considering launching rosbridge in autowarefoundation as well.

**Comment by kenji-miyake** [comment]
@yabuta @isamu-takagi Could you answer this? :pray:

**Comment by kenji-miyake** [comment]
(Internal discussion thread: https://star4.slack.com/archives/C02JH3FME8N/p1643333178207500)

**Comment by yabuta** [comment]
@shmpwk rosbridge is used to communicate with the TIER IV cloud system and TIER IV tools as well as web controller.
Since rosbridge communicates with nodes other than web controller, we moved rosbridge launcher to autoware_api_launch, which handles external integration.

**Comment by shmpwk** [comment]
@yabuta Thank you for the information, 
My understanding is that currently autowarefoundation does not use rosbridge in autoware_api_launch but in the future it might use it. 
So should autowarefoundation launch rosbridge in autoware_api_launch as well as this PR?

**Comment by yabuta** [comment]
@shmpwk 
Are autowarefoundation using autoware_api_launch?
If so, I think that it is good to add rosbridge launcher to autoware_api_launch.
However, I think it would be better to write how to launch rosbridge in DOCS of web_controller for future users who don't use autoware_api_launch.

**Comment by shmpwk** [comment]
@yabuta 
In autowarefoundation, there is 
https://github.com/autowarefoundation/autoware.universe/tree/main/launch/tier4_autoware_api_launch.
However considering not launching rosbridge in tier4_autoware_api_launch, nobody uses it for now (?)
But it must be used when FOA accesses autoware through rosbridge.

Thus, I add rosbridge launcher to autoware_api_launch and also add docs in web_controller.

**Comment by yabuta** [comment]
@shmpwk 
Thank you.
I think that's a good idea.

**Pull request merged**